service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DocmosisTemplates.java

```diff
@@ -1,6 +1,7 @@
 package uk.gov.hmcts.reform.fpl.enums;
 
 public enum DocmosisTemplates {
+    C110A("FL-PLW-APP-ENG-00095.doc", "case_submission_c110a.pdf"),
```

Comment:
case_submission_c110a.pdf seems like an odd name

Reply:
This name is not used anywhere. We are actually generating file name in https://github.com/hmcts/fpl-ccd-configuration/pull/1133/files#diff-69b9a7515cf1144e3544b72648e82810R36

Reply:
Might be nice to leave a comment like the ORDER constant